SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1090,31 +1089,37 @@ void USpatialActorChannel::UpdateSpatialPosition()
 		return;
 	}
 
-	SendPositionUpdate(Actor, EntityId, NewSpatialActorPosition);
+	SendPositionUpdate(NewSpatialActorPosition);
 
 	if (APlayerController* PlayerController = Cast<APlayerController>(Actor))
 	{
 		if (APawn* Pawn = PlayerController->GetPawn())
 		{
-			SendPositionUpdate(Pawn, NetDriver->PackageMap->GetEntityIdFromObject(Pawn), NewSpatialActorPosition);
+			if (USpatialActorChannel* Channel = Cast<USpatialActorChannel>(Connection->FindActorChannelRef(Pawn)))
```

Comment:
Is the reason we're doing this as we tie playercontroller/pawn position and need to also update the pawn's actorchannel?

if answer isn't obvious with context on this, might also be good to have a comment

Reply:
If the pawn were to become unpossessed, this information stored on it's actor channel would be incorrect/stale.